build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException-BuildTarget.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
Why do you have to source include anything in the tests in the first place? That seems like the wrong thing to do to begin with.